Capstone project

Mahdi-Niazi opened the pull request:
In this milestone I:
Add Mobile version of the site
Add Desktop version of the site
Add Responsiveness to the site
Add dynamic JS code to the site
Add Video link to the readme file
Add deploy link to the readme file

Mahdi-Niazi committed:
Add HTML linters
Add Stylelints
Add eslinter

Mahdi-Niazi committed:
Add required images
Add required fonts

Mahdi-Niazi committed:
Add responsive menu

Mahdi-Niazi committed:
Add introduction section
Add required html elements
Add required css properties

Mahdi-Niazi committed:
Add home page to the website.

Mahdi-Niazi committed:
Add about page with its sections

Mahdi-Niazi committed:
Add about page to the webstie
Add mobile responsivenees to about page

Mahdi-Niazi committed:
Add dynamic to feature players

Mahdi-Niazi committed:
Fix menu background in mobile version

Mahdi-Niazi committed:
Fix header of about page

Mahdi-Niazi committed:
Fix issues with website responsiveness

Mahdi-Niazi committed:
Fix stylelint errors

Mahdi-Niazi committed:
Add deploy link to readme file

Mahdi-Niazi committed:
Add video link to the website

Mahdi-Niazi committed:
Fix navbar issues

shella12 commented on index.html:
```diff
@@ -0,0 +1,225 @@
+    </nav>
+    <nav class="navbar main-nav navbar-expand-lg">
+        <div class="container">
+            <a class="navbar-brand d-none d-sm-flex" href="index.html">
```
- [x] [OPTIONAL] I would highly recommend you to make your hamburger menu cover the whole screen when it's visible to focus the user’s attention on the menu items and avoid distractions from other elements on the page. A full-screen hamburger menu can create a sense of immersion and engagement, as well as make the menu items more readable and accessible

Your design:

<img width="290" alt="Screenshot 2023-06-28 at 10 32 40 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/a3a064da-5ea2-488f-aa60-187950d8edf0">

Suggested design:

<img width="355" alt="Screenshot 2023-06-28 at 11 44 05 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/2bdfe8d1-5746-4f18-901b-03ff5165ce46">

shella12 commented on about.html:
```diff
@@ -0,0 +1,182 @@
+    </nav>
+    </header>
+    <section class="intro-sec">
+        <div class="container-fluid">
+            <div class="container header-container d-flex flex-wrap align-content-center">
```
- [x] There is too much padding around the elements, please reduce the paddings of `container-fluid` and `container` in your website to match the design. This is important because it ensures consistency and alignment with the design specifications

Note: This suggestion applies to all of your sections with `container-fluid` and `container`
classes


|Your Design|Behance Design|
|-----|----|
|<img width="289" alt="Screenshot 2023-06-28 at 10 32 29 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/6cb706ce-e978-470f-94df-4e484aa95202">|<img width="387" alt="Screenshot 2023-06-28 at 12 41 52 PM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/af9c2807-839e-43a2-9bbd-fa886f1a85c6">|
|<img width="359" alt="Screenshot 2023-06-28 at 10 35 12 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/470f9cd1-e5e1-4bd4-82e4-35094debff51">|<img width="452" alt="Screenshot 2023-06-28 at 10 39 59 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/ba6e0b35-0d3e-45bb-9755-bdbf07ba857a">|
|<img width="367" alt="Screenshot 2023-06-28 at 10 35 34 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/9627b75c-73ac-475b-b942-4d0688bd540f">|<img width="453" alt="Screenshot 2023-06-28 at 10 39 50 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/8b5b676b-17f2-48ed-b514-c3e62ae5e350">|
|<img width="359" alt="Screenshot 2023-06-28 at 10 41 42 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/e3e48e90-4d2a-4eb1-bedd-f9a1d558665a">|<img width="451" alt="Screenshot 2023-06-28 at 10 41 10 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/95a82950-3785-4e98-9873-30309fb55dac">|
|<img width="356" alt="Screenshot 2023-06-28 at 10 36 04 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/88510299-ca3a-44d5-a315-a5f1811aebe3">|<img width="456" alt="Screenshot 2023-06-28 at 10 42 06 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/b01f46b7-0dc3-4fc2-9e55-e8de88a859b3">

shella12 commented on assets/JS/script.js:
```diff
@@ -0,0 +1,70 @@
+            </div>
+        </div>      
+    </div>
+  </div>      
```
- [ ] Great job on using array objects to organise players data however please adjust the sizes of your player's images to make sure your implementation matches the design

|Figma | Yours|
|--|--| 
|<img width="451" alt="Screenshot 2023-06-28 at 10 41 10 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/a98e7f7e-5165-4495-b328-d1f96164dbda">|<img width="354" alt="Screenshot 2023-06-28 at 10 31 41 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/cfb3f3ad-6702-4acd-b36b-54c0a19e1bda">|

shella12 commented on about.html:
```diff
@@ -0,0 +1,182 @@
+            <div class="container header-container d-flex flex-wrap align-content-center">
+                <div class="row justify-content-center m-3">
+                    <div class="col-md-12 intro-p abt">
+                        <h1>Afghanistan National Team 2023 Afghanistan Football Federation 1922 </h1>
+                    </div>
+                </div>
+
+                <div class="txt-border"></div>
+
+                <div class="row justify-content-center m-3">
+                    <div class="col-md-12 header-description abt-txt">
+                        <p>
+                            The Afghanistan national football team 
+                            is the national football team of Afghanistan and is controlled by the
+                            Afghanistan Football Federation. Founded in 1922, they played their first
+                            international game against Iran in Kabul, 1941. Afghanistan then joined
+                            FIFA in 1948 and the AFC in 1954, as one of the founding members.
+                            They play their home games at the Ghazi National Olympic Stadium in Kabul, 
+                            the capital of Afghanistan. In 2013, Afghanistan won the 2013 SAFF Championship
+                            and earned the "FIFA Fair Play Award". Afghanistan has never qualified for the AFC Asian Cup.
```
- [ ] I would suggest reducing padding and sizes of images in your events section in about page to match the provided design

|Behance Design| Your's|
|---|---|
|Mobile screen:|---|
|<img width="453" alt="Screenshot 2023-06-28 at 10 39 50 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/b871697f-1fa5-4624-bda2-f38079aa9f76">|<img width="941" alt="Screenshot 2023-06-28 at 10 33 06 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/e528f066-7c01-438d-8965-1e659c3b5fb7">|
|Desktop Screen:|---|
|<img width="866" alt="Screenshot 2023-06-28 at 10 39 13 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/837cf5af-7ae3-42b2-8f56-acaae5b8d72c">|<img width="1426" alt="Screenshot 2023-06-28 at 10 39 24 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/dfa5443d-11e4-4978-be7e-24bf2f922b5e">|

shella12 commented on index.html:
```diff
@@ -0,0 +1,225 @@
+            </a>
+            <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#collapsibleNavbar">
+                <span class="navbar-toggler-icon"></span>
+            </button>
+            <div class="collapse navbar-collapse justify-content-end collapse-main" id="collapsibleNavbar">
+                <ul class="navbar-nav">
+                    <li class="nav-item">
+                        <a href="#" class="nav-link selected"> Home </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#programs" class="nav-link"> Programs </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#players" class="nav-link"> Players </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="#partners" class="nav-link"> Partners </a>
+                    </li>
+                    <li class="nav-item">
+                        <a href="about.html" class="nav-link">About</a>
+                    </li>
+                    <li class="nav-item compaign">
+                        <a href="#" class="nav-link">CC Campaign</a>
+                    </li>
```
- [ ] Please align the content of program section elements to match the design


|Your's| Behance design|
|---|---|
|<img width="283" alt="Screenshot 2023-06-28 at 10 32 09 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/dbbf3041-4d1c-4547-8d91-58e5d81ba17d">|<img width="448" alt="Screenshot 2023-06-28 at 10 40 28 AM" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/44798044/f138a386-b6c2-4d77-8a79-98565ae378a3">|

shella12 reviewed (changes requested):
Hi @Mahdi-Niazi, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!

<img src="https://media1.giphy.com/media/Jq824R93JsLwZCaiSL/giphy.gif"/>

## Highlights
- Deployed site ✅ 
- Professional demo video ✅   
- Great concept ✅ 
- Good use of semantic tags ✅ 

## Required Changes ♻️

- [ ] Great demo video however please make sure to talk about a piece of code you are most proud of in your video, as it is the requirement of the project 
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mahdi-Niazi commented:
@shella12 Thank you for your valuable feedbacks I'll do my best to apply your comments.

Mahdi-Niazi committed:
Fix navbar responsive
Fix introduction section padding and margins
Fix about page logo section
Fix footer for all pages
Fix main program section
Fix feature players section

Mahdi-Niazi committed:
Update video link in readme file

Mahdi-Niazi committed:
Fix stylelint errors

Mahdi-Niazi committed:
Update video link in readme file

mckent05 commented on assets/JS/script.js:
```diff
@@ -0,0 +1,87 @@
+  bars.classList.remove('active');
+}));
+
+const playerDetails = [
+  {
+    fullName: 'Islam Amiri',
+    position: 'Centre-back',
+    description: 'Amiri made his debut in the 2005 SAFF Gold Cup against Maldives.',
+    playerImage: '/assets/images/guest-player-1.png',
+    altText: 'Islam Amiri',
+  },
+  {
+    fullName: 'Faysal Shayesteh',
+    position: 'Attacking midfielder',
+    description: 'Born in Kabul, Shayesteh moved to the Netherlands and played in the youth academy',
+    playerImage: '/assets/images/guest-player-2.png',
+    altText: 'Faysal Shayesteh',
+  },
+  {
+    fullName: 'Abassin Alikhil',
+    position: 'Defensive midfielder',
+    description: 'Abassin Alikhil is an Afghan professional footballer who plays as a defensive midfielder.',
+    playerImage: '/assets/images/guest-player-3.png',
+    altText: 'Abassin Alikhil',
+  },
+  {
+    fullName: 'Mustafa Hadid',
+    position: 'Right back',
+    description: 'Born in Kabul, Hadid moved to Hamburg with his family in 2016 escaping the Third Afghan Civil War (1996–2001).',
+    playerImage: '/assets/images/guest-player-4.png',
+    altText: 'Mustafa Hadid',
+  },
+];
```
**OPTIONAL** - According to the design template, your speaker container should contain information about 6 different speakers, please add more speakers to your array 👍 👍

mckent05 commented on about.html:
```diff
@@ -0,0 +1,179 @@
+                    </div>
+                </div>
+                <div class="row m-lg-5 g-3">
+                    <div class="col-md-6 col-sm-12">
+                        <div class="history-bg-1">
+                            <h2 class="text-white">2003</h2>
+                            <p class="text-white">Afghanistan National Team at Ghazi Studium</p>
+                        </div>
+                    </div>
+                    <div class="col-md-6 col-sm-12">
+                        <div class="history-bg-2">
+                             <h2 class="text-white">2023</h2>
+                            <p class="text-white">Afghanistan National Team at Ghazi Studium</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="abt-footer">
+        <div class="container-fluid">
+            <div class="container">
+                <div class="row p-lg-5 ">
+                     <div class="footer d-flex justify-content-center align-items-center">
+                        <div class="col-md-3 col-sm-6">
+                            <img src="assets/images/Logo.png" alt="Afghan Team Logo" width="100px" height="120px">
+                        </div>
+                        <div class="col-md-9 col-sm-6">
+                           <h5>
+                               The Afghanistan national football team 
+                            </h5>
+                            <p>
+                            Afghanistan Football Federation. Founded in 1922, they played their first
+                           </p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </section>
```
**OPTIONAL** - For the mobile version, the partner section is missing on this page, please fix this. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="250" alt="jtjrj" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/73607512/34ecdba2-76db-4d68-ba16-37ab7d851f02">

Project requirement:
<img width="310" alt="wrawrhar" src="https://github.com/Mahdi-Niazi/Capstone-Project-HTML-CSS-JS/assets/73607512/e22790e0-e83d-449f-9398-e5acd810dd8f">

mckent05 reviewed (approved):
Hi @Mahdi-Niazi, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
![approved](https://user-images.githubusercontent.com/73607512/207264357-1b05ce6a-95f2-4d31-9c26-888c6ddfa124.gif)


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 
- Good job working on your first Microverse capstone, keep going 🚀 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
